node/Gopkg.lock

```diff
@@ -1,25 +1,6 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
-[[projects]]
```

Comment:
Shouldn't check in this file, unless we've explicitly changed the toml file to update the dependencies. We're not quite doing this correctly right now, but most time I just roll back these changes from the update, before committing (since they may cause problems with other developers setups).